mailbox/store/src/test/java/org/apache/james/mailbox/store/mail/model/impl/PropertyBuilderTest.java

```diff
@@ -26,12 +26,27 @@
 import org.apache.james.mailbox.store.mail.model.Property;
 import org.junit.jupiter.api.Test;
 
+import java.util.*;
```

Comment:
```suggestion
import java.util.Arrays;
import java.util.Collections;
import java.util.List;
import java.util.Map;
```

Reply:
Is it preferred to have explicit imports?